packages/hoppscotch-backend/src/mailer/mailer.service.ts

```diff
@@ -42,6 +46,8 @@ export class MailerService {
     to: string,
     mailDesc: MailDescription | UserMagicLinkMailDescription,
   ) {
+    if (this.configService.get('INFRA.MAILER_SMTP_ENABLE') !== 'true') return;
```

Comment:
We can throw an error here right saying "SMTP_NOT_ENABLED" ?

Reply:
Since we using event emit, throwing errors is not effective.